software/acpid.txt

```diff
@@ -0,0 +1,155 @@
+
+- Acpid2                                                                   [2.0]
+    - Finding Events                                                       [2.1]
+    - Handling Events                                                      [2.2]
+
+
+[0.0] Usage
+________________________________________________________________________________
+
+KISS Linux offers two options for acpid management: busybox acpid, which is
+installed by default, and acpid2, which can be installed from the acpid
+package. For both acpid implimentations, you will need to enable a few
+ACPI-related kernel options and optionally the runit service to manage it.
+
+To enable the acpid runit service run:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ ln -s /etc/sv/acpid/ /var/service/                                       |
+|   $ sv up acpid                                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Whenever changes are made, restart acpid for them to take effect.
+
++--------------------------------------------------------------------------+
+|                                                                          |
+|   $ sv restart acpid                                                     |
+|                                                                          |
++--------------------------------------------------------------------------+
+
+
+    [0.1] Kernel Configuration
+    ____________________________________________________________________________
+
+    For now, check the Gentoo Wiki for details. [0] The important information
+    will be added to this page soon.
+
+
+[1.0] Busybox acpid
+________________________________________________________________________________
+
+Busybox provides a very limited acpid program, that only works well for laptop
+lid events. Everytime the lid is closed, acpid will execute /etc/acpid/LID/00000080.
+Make sure this file is executable The following example can be used to suspend
+your laptop to RAM.
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   #!/bin/sh                                                                  |
+|                                                                              |
+|   printf mem > /sys/power/state                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+[2.0] acpid2
+________________________________________________________________________________
+
```

Comment:
A description about acpid2 would be nice. :+1:

Reply:
More information about `acpid2`. :+1:

Reply:
I'm fairly certain it's just a more expansive version. It handles more events and is configured differently. Not sure what else you're looking for